.gitignore

```diff
@@ -82,4 +82,6 @@ buck-out/
 web-build/
 dist/
 
-# @end expo-cli
\ No newline at end of file
+# @end expo-cli
+
+yarn.lock
```

Comment:
this is committed on purpose.
sometimes is needed for `vendoring` to make sure the CI pipelines can build the app as expected.

Reply:
Removed on https://github.com/TERITORI/teritori-dapp/pull/637